hfie/endpoint.py

```diff
@@ -86,12 +86,27 @@ def update(
 
 
 @app.command()
-def delete(name: str = typer.Argument(..., help="Endpoint name")):
+def delete(
+    name: str = typer.Argument(..., help="Endpoint name"),
+    force: bool = typer.Option(
+        False, help="Force deletion without asking user confirmation"
+    ),
+):
     """
     Delete an endpoint
     """
-    r = requests.delete(f"{settings.endpoint_url}/{name}", headers=headers)
-    typer.echo(r.json())
+    if not force:
+        decision = input(
```

Comment:
Can we implement the input with `typer.confirm` e.g. https://github.com/ivyleavedtoadflax/remote.py/blob/4ce8914ead56947ecd03d044e54adcea8a62e961/remotepy/instance.py#L333

```
        confirm = typer.confirm(
            f"Are you sure you want to stop instance {instance_name}?",
            default=True,
        )
```

Reply:
722a26b